code/game/objects/map_metadata/hill_3234.dm

```diff
@@ -19,15 +19,15 @@
 	age = "1988"
 	ordinal_age = 7
 	faction_distribution_coeffs = list(RUSSIAN = 0.3, ARAB = 0.7)
-	mission_start_message = "<font size=4>All factions have <b>5 minutes</b> to prepare before the battle begins!<br>The <font color = 'red'>Soviets</font> will win if they hold out the hill for <b>40 minutes</b>.<br>The <b><font color = 'black'>Mujahideen</font></b> will win if they manage to capture the radio station on the top of the hill!</font>"
+	mission_start_message = "<font size=4>All factions have <b>8 minutes</b> to prepare before the battle begins!<br>The <font color = 'red'>Soviets</font> will win if they hold out the hill for <b>40 minutes</b>.<br>The <b><font color = 'black'>Mujahideen</font></b> will win if they manage to capture the radio station on the top of the hill!</font>"
 	faction1 = RUSSIAN
 	faction2 = ARAB
 	valid_weather_types = list(WEATHER_NONE, WEATHER_WET)
 	songs = list(
 		"Kino - Gruppa Krovi (Blood Group):1" = "sound/music/gruppakrovi.ogg",)
 	gamemode = "Siege"
 	artillery_count = 3
-	grace_wall_timer = 3000
+	grace_wall_timer = 4800 // 8 minutes.
```

Comment:
why the increase? they already have the insane advantage of it being a single lane map

Reply:
![image](https://github.com/Civ13/Civ13/assets/131271192/3b3634b7-1ac8-4d67-a85b-1f1e3fbca22f)

Three lane map.
I tried to defend it and barely got to even start placing mines, (walking all the way over there.) before grace went up.

Reply:
I was lenient on the amount of minutes as we should really use an exponential grace-period based on pop

Reply:
they changed the final wall

Reply:
My branch is up to date.